dnf5/commands/download/download.cpp

```diff
@@ -115,6 +118,12 @@ void DownloadCommand::set_argument_parser() {
             return true;
         });
 
+    auto urlallmirrors = parser.add_new_named_arg("all-mirrors");
```

Comment:
Could you please move this block just after `auto url = parser.add_new_named_arg("url");` block? So that it is handled in the same order as urlallmirrors_option is initialized just after url_option?